lib/kernel/test/gen_sctp_SUITE.erl

```diff
@@ -2339,6 +2374,26 @@ do_simple_sockaddr_send_recv(#{family := _Fam} = SockAddr, _) ->
     ok.
 
     
+%% Test the sockaddr version of connectx_init/4.
+t_simple_local_sockaddr_in_connectx_init(Config) when is_list(Config) ->
+    Localhost = {127,0,0,1},
+    {ok,Sb} = gen_sctp:open(),
+    {ok,Pb} = inet:port(Sb),
+    {ok,Sa} = gen_sctp:open(),
+    SockAddr = #{family   => inet,
+                 addr     => Localhost,
+                 port     => Pb},
+    {ok, A1} = gen_sctp:connectx_init(Sa, [SockAddr], []),
```

Comment:
On OpenIndiana `gen_sctp:connectx_init/3` here returns `{error,enotsup}` which does not match and the test fails.
I think that should cause the test case to be skipped instead.

Reply:
Agreed. In my comment I meant that the test should _be changed_ to handle that case. Sorry for the confusion